### ISSUE TITLE
Behavior of parse_number 

### ISSUE THREAD

**Issue by PMassicotte** [tidyverse/readr#768]
I found that `parse_number()` was not always working as I was expecting.

``` r
                            
number <- "-4.555906344E-06"
                            
readr::parse_number(number) 
#> [1] -4.555906
                            
as.numeric(number)          
#> [1] -4.555906e-06
```
If `parse_number` is working as intended, is it better to just use `as.numeric` in that context?

Thank you,
Phil

**Comment by batpigandme** [comment]
HI @PMassicotte,

I'm not getting the same result. What version of readr are you using? Below is with `readr_1.1.1.9000` (i.e. the dev version). 

``` r
number <- "-4.555906344E-06"
readr::parse_number(number)
#> [1] -4.555906e-06
```

Created on 2017-12-22 by the [reprex
package](http://reprex.tidyverse.org) (v0.1.1.9000).

**Comment by PMassicotte** [comment]
Hi @batpigandme, thank you for the reply. I am using the latest published version on CRAN (1.1.1). I can post the `SessionInfo()` when I am back to the office.

**Comment by batpigandme** [comment]
@PMassicotte I'd just suggest installing the dev version, if you can, since it seems like it's been fixed. If _that_ doesn't work, then sessionInfo might be in order 👍.

**Comment by batpigandme** [comment]
Oh, yes, it's patched from issue #684 
See the bullet in https://github.com/tidyverse/readr/blob/master/NEWS.md
